ROMFS/px4fmu_common/init.d/rc.vehicle_setup

```diff
@@ -32,6 +32,15 @@ then
 	. ${R}etc/init.d/rc.rover_apps
 fi
 
+#
+# Spapcecraft setup.
+#
+if [ $VEHICLE_TYPE = sc ]
```

Comment:
Can we spell it out everywhere instead?

```suggestion
if [ $VEHICLE_TYPE = spacecraft ]
```

Reply:
Sure, I'll do it in the rate controller PR (I think that's where these get in)